relay-server/src/metrics_extraction/event.rs

```diff
@@ -1308,7 +1308,7 @@ mod tests {
     #[test]
     fn test_app_start_cold_outlier() {
         let metrics = extract_span_metrics_mobile("app.start.cold", 181000.0);
-        assert!(metrics.is_empty());
+        assert_eq!(metrics.len(), 1);
```

Comment:
Validating there's one metric without knowing which one is not very valuable, and we should assert on the expected metric instead (with e.g. a snapshot).

Reply:
I changed the test to look for the specific metirc name.